src/lib/requests/common.js

```diff
@@ -34,10 +34,6 @@ class HTTPResponseError extends Error {
     toString() {
         return util.inspect(this[respErrSym]);
     }
-
-    toJSON() {
```

Comment:
Won't it break some other packages?  I see we export `HTTPResponseError` from this module, so IDK how and where it's used

Reply:
The code looks weird, but I checked where `HTTPResponseError` is used using
https://github.com/search?q=org%3Apm4ml%20HTTPResponseError&type=code
and only found a reference in https://github.com/pm4ml/mojaloop-payment-manager-system-of-record/blob/master/src/middlewares.ts

If you think it is dangerous to change this, I can revert it. The problem is that error logging is inconsistent (i.e. this specific error is logged in a weird stringified way) and may cause difficulties in places like loki.

Reply:
as i said above:

>  ...IDK how and where it's used

Reply:
I am not sure, so returned the original code.